packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -128,6 +129,14 @@ export interface DatabaseClusterProps {
    * @default - Retain cluster.
    */
   readonly deleteReplacePolicy?: DeletionPolicy
+
+  /**
+   * The interval, in seconds, between points when Amazon RDS collects enhanced
+   * monitoring metrics for the DB instances.
+   *
+   * @default no enhanced monitoring
+   */
+  readonly monitoringInterval?: number;
```

Comment:
Please annotate this property with a unit. Recommended: `monitoringIntervalSec`.

Although @RomainMuller is currently working on a `Duration` class which will be the recommended practice going forward.

Reply:
could be also adapted in `DatabaseInstance` then.

https://github.com/awslabs/aws-cdk/blob/e0718ef7cc26798ec2401451be9d7b7894f0b8bc/packages/%40aws-cdk/aws-rds/lib/instance.ts#L375

(breaking)

Reply:
I renamed that property in `DatabaseClusterProps` and in `DatabaseInstanceNewProps` as suggested.
I also added the _Breaking_ advise and rebased to _master_!

Reply:
@rix0rrr I've seen that the `Duration` class has landed on master branch so I updated this PR with those required changes. Let me know if you need anything else!